src/components/Pagination/PaginationPage.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+
+type Props = {
+  page: number;
+  currentPage: number;
+  onClick: (page: number) => void;
+};
+
+export const PaginationPage = ({ page, currentPage, onClick }: Props) => {
+  return (
+    <li
+      key={page}
```

Comment:
you don't need to add key here [[more info]](https://legacy.reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys)